software/controller/lib/debug/var_cmd.cpp

```diff
@@ -71,80 +73,82 @@
   // <fmt>  - variable length format string
   // <help> - variable length format string
   // The strings are not null terminated.
-  int nLen = static_cast<int>(strlen(var->GetName()));
-  int fLen = static_cast<int>(strlen(var->GetFormat()));
-  int hLen = static_cast<int>(strlen(var->GetHelp()));
+  size_t name_length = strlen(var->GetName());
+  size_t format_length = strlen(var->GetFormat());
+  size_t help_length = strlen(var->GetHelp());
 
-  if (nLen > 255)
-    nLen = 255;
-  if (fLen > 255)
-    fLen = 255;
-  if (hLen > 255)
-    hLen = 255;
```

Comment:
Hmmmm... this feels hacky for some reason. Maybe this is unnecessarily complex? Are we really saving valuable bandwidth by not having these just be of some static length big enough to fit whatever we might put in there?

Reply:
I think the only purpose of this is to prevent a timeout rather than save bandwidth.
I also believe the max response length is already covering this, so in short, these may very well be pointless.